proposal.md

```diff
@@ -10,9 +10,9 @@ Our objective is to determine whether or not these methods can be trained with s
 
 ## Aims
 
-**Aim 1: Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data from the HCA.**
+**Aim 1: Develop proof-of-concept unsupervised deep learning methods for single cell transcriptomic data from the HCA.**
 
-**Aim 2: Evaluate the extent to which low-dimensional representations of single cell data learned by various methods can be used to decompose bulk tissues in the context of rheumatic diseases.**
+**Aim 2: Generate a benchmark dataset of harmonized public data to evaluate the extent to which HCA cell types capture disease states for multiple rheumatic diseases.**
```

Comment:
are we doing `cell types` or `cell-types`? I used `cell-types` in aims 1 and 3. Either way, we should be consistent